packages/iTwinUI-react/.eslintignore

```diff
@@ -0,0 +1,13 @@
+**/*.json
```

Comment:
couldnt move this file, so it shows only as renamed? same with `.eslintrc.js`

Reply:
not sure what you mean

Reply:
Now it shows that you deleted the one we have and this is a new one. But I assume, this is the same file, just moved.
So it would be better to review when you see actual changes in the file (if there are any) versus seeing this as new file and then manually going to compare with the old one.

Reply:
Well, I can't change the way git detects a file as changed vs added.

What you could do is use some external diff checker, or even just VSCode (create new folder, run `git init`, create a file, put old contents there, stage the file, then put new contents there, and check diff in the source control). It's not nice but it will show you the changes (there are some).